frontend/src/themes/theme.ts

```diff
@@ -16,7 +16,7 @@ declare module "@mui/material/Button" {
 }
 
 // Create core theme so we can access spacing etc. when customizing components
-const coreTheme = createTheme({
+export const coreTheme = createTheme({
```

Comment:
exporting the coreTheme so that the signup theme and other themes can depend on it.